release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -56,6 +56,21 @@ link:https://access.redhat.com/support/policy/updates/openshift[Red Hat OpenShif
 
 This release adds improvements related to the following components and concepts.
 
+[id="ocp-4-6-web-console"]
+=== Web console
+
+[id="ocp-4-6-web-console-column-management"]
+==== Column management
+
+A *Manage columns* icon image:manage-columns.png[title="Manage Columns icon"] is
+now added to some resources pages. When you click on the icon, default column
```

Comment:
Is there an example page we can provide here since the new icon is available on _some_ resources pages?

s/is  now added to some resources pages, for example the <example> page.

Reply:
For example Pods page